packages/client/src/contracts/StreamStorageRegistry.ts

```diff
@@ -35,7 +37,7 @@ interface NodeQueryResult {
 export class StreamStorageRegistry {
 
     private streamStorageRegistryContract?: StreamStorageRegistryContract
-    private readonly streamStorageRegistryContractsReadonly: StreamStorageRegistryContract[]
+    private readonly streamStorageRegistryContractsReadonly: StreamStorageRegistryContract
```

Comment:
Rename (to singular)
```suggestion
    private readonly streamStorageRegistryContractReadonly: StreamStorageRegistryContract
```